services/src/main/java/org/keycloak/utils/CRLUtils.java

```diff
@@ -51,15 +51,15 @@ public final class CRLUtils {
      * @throws GeneralSecurityException if some error in validation happens. Typically certificate not valid, or CRL signature not valid
      */
     public static void check(X509Certificate[] certs, X509CRL crl, KeycloakSession session) throws GeneralSecurityException {
-        if (certs.length < 2) {
-            throw new GeneralSecurityException("Not possible to verify signature on CRL. X509 certificate doesn't have CA chain available on it");
+        if (certs == null || certs.length < 1) {
+            throw new GeneralSecurityException("Not possible to verify signature on CRL because no certificate chain passed.");
```

Comment:
```suggestion
            throw new GeneralSecurityException("Not possible to verify signature on CRL because no certificate chain was passed.");
```

Reply:
Done!